parts/k8s/addons/antrea.yaml

```diff
@@ -378,23 +440,29 @@ spec:
         image: {{ContainerImage "antrea-controller"}}
         name: antrea-controller
         ports:
-        - containerPort: 443
+        - containerPort: 10349
+          name: api
           protocol: TCP
         readinessProbe:
           failureThreshold: 5
           httpGet:
             host: 127.0.0.1
             path: /healthz
-            port: 443
+            port: api
             scheme: HTTPS
           initialDelaySeconds: 5
           periodSeconds: 10
           timeoutSeconds: 5
+        resources:
+          requests:
+            cpu: 200m
```

Comment:
These resource requests would ideally be part of the addon's configuration, not hard-coded here. Maybe that's a separate PR.

Reply:
Done